.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: csharp
+solution: MoreLinq.sln
+mono: latest
+dist: trusty
+sudo: required
+dotnet: 1.0.0-preview2-003131
+
+install:
+  - dotnet restore
+
+script:
+  - dotnet build -c Debug MoreLinq
```

Comment:
Not really. I could add both if you want to. I did not manage to test the Release configuration though. `dotnet` insisted on trying to use the debug build.